graphql_client_codegen/src/lib.rs

```diff
@@ -137,24 +143,34 @@ pub fn generate_module_token_stream(
     Ok(modules)
 }
 
-#[derive(Debug, Error)]
+#[derive(Debug)]
 enum ReadFileError {
-    #[error(
-        "Could not find file with path: {}\
-        Hint: file paths in the GraphQLQuery attribute are relative to the project root (location of the Cargo.toml). Example: query_path = \"src/my_query.graphql\".",
-        path
-    )]
-    FileNotFound {
-        path: String,
-        #[source]
-        io_error: io::Error,
-    },
-    #[error("Error reading file at: {}", path)]
-    ReadError {
-        path: String,
-        #[source]
-        io_error: io::Error,
-    },
+    FileNotFound { path: String, io_error: io::Error },
+    ReadError { path: String, io_error: io::Error },
+}
+
+impl Display for ReadFileError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            ReadFileError::FileNotFound { path, .. } => {
+                write!(f, "Could not find file with path: {}\
```

Comment:
This is missing a space after the filepath.

```suggestion
                write!(f, "Could not find file with path: {} \
```